app/preprints/-components/submit/preprint-state-machine/template.hbs

```diff
@@ -3,13 +3,17 @@
     onClickStep=this.onClickStep
     onNext=this.onNext
     onSave=this.onSave
+    preprint=this.preprint
+    nextButtonIsDisabled=this.nextButtonisDisabled
 
     getTitleAndFileType=this.getTitleAndFileType
     getMetadataType=this.getMetadataType
     getAuthorAssertionsType=this.getAuthorAssertionsType
     getSupplementsType=this.getSupplementsType
     getReviewType=this.getReviewType
 
+    validateTitleAndFile=this.validateTitleAndFile
```

Comment:
Instead of yielding an action for each section, we could yield a `validateSecionByIndex` action and the `statusFlowIndex` property.